packages/iris-grid/src/CommonTypes.tsx

```diff
@@ -0,0 +1,9 @@
+import { ModelIndex } from '@deephaven/grid';
+import AggregationOperation from './sidebar/aggregations/AggregationOperation';
+import { AdvancedFilter, QuickFilter } from './IrisGrid';
```

Comment:
`AdvancedFilter`, `QuickFilter`, `AdvancedFilterOptions`, `AggregationOperation`, basically all those types should come into here, as they are all used across many files within iris-grid.

Right now this is importing from IrisGrid, which imports from AdvancedFilterCreator, which imports from AdvancedFilterCreatorSelectValue, which imports from CommonTypes, got a cycle.

Pretty much all the types currently defined and exported in IrisGrid.tsx should just be in CommonTypes